### ISSUE TITLE
failure to play 3 player splitscreen with 1 friend joining us via online (steam)

### ISSUE THREAD

**Issue by RoseGenki** [WinterSquire/AlphaRing#26]
so we tried to play cursed halo 1 with 4 people yet it keeps disconnecting us before it finishes loading, even when we try regular halo 2 it does the same, we know it's not internet problem so what could be causing this? it loads fine with me and my 2 mates splitscreening it but we just can't get our other friend in too :( any advice?

**Comment by WinterSquire** [comment]
Right now you can't do online Co-Op with splitscreen on in H1/H2.
If you want to play with another person online, currently the solution is using [Parsec](https://parsec.app/).

**Comment by dylanmoh** [comment]
I'm sorry am I reading this correctly? You can do 4 players for Halo 1 campaign??? Does this also work for 2 and their anniversary editions?

**Comment by WinterSquire** [comment]
Yes, you can do 4 players local Co-Op in all games. I made some improvements to the game.

**Comment by RoseGenki** [comment]
> Right now you can't do online Co-Op with splitscreen on in H1/H2. If you want to play with another person online, currently the solution is using [Parsec](https://parsec.app/).

problem is it also disconnected us when trying the exact same with reach which was shown in the video to work? odst as well but we didn't try that. i found this from generalkidds video

**Comment by dylanmoh** [comment]
> Yes, you can do 4 players local Co-Op in all games, but only works correctly in classic graphics. I made some improvements to the game.



> Yes, you can do 4 players local Co-Op in all games, but only works correctly in classic graphics. I made some improvements to the game.

You are doing God's work. Thank you!!! Would the anniversary editions eventually be able to get 4 players as well? Or is this currently an impossibility?
